app/uk/gov/hmrc/transitmovementsrouter/controllers/MessagesController.scala

```diff
@@ -110,6 +110,9 @@ class MessagesController @Inject() (
         persistenceResponse <- persistenceConnector
           .postObjectStoreUri(movementId, messageId, messageType, ObjectStoreURI(objectSummary.location.asUri))
           .asPresentation
+        _ = pushNotificationsConnector
+          .postForLargeMessages(movementId, persistenceResponse.messageId, null, ObjectStoreURI(objectSummary.location.asUri))
```

Comment:
Here you need to pass the messageType instead of ``` null ```

Reply:
Okay